revolv/project/models.py

```diff
@@ -291,17 +325,18 @@ def has_owner(self, ambassador):
 
     def approve_project(self):
         self.project_status = Project.ACTIVE
-        self.start_date = datetime.date.today()
+        if self.start_date is None:
```

Comment:
Good call. Would we need to do something like reset the end_date when the project is completed?